### PR TITLE
Add links to lisk service related content

### LINKED ISSUE #1836: Add links to Lisk Service related content
### Motivation
Lisk service-related content is going through an update, and once the update is complete, we need to link all the relevant content with each other to aid readers in learning about Lisk Service.

### Description

- Once the content is ready, add the following link to the `aggregate-blockchain-data/index.adoc`
[Connecting to endpoints guide](https://lisk.com/documentation/beta/integrate-blockchain/connecting-endpoints.html)
- Add the link to the new page **How to extend indexing of Lisk Service** to `aggregate-blockchain-data/index.adoc`
- Add links to the service guides mentioned above, [here](https://lisk.com/documentation/run-blockchain/index.html#lisk-service-nodes) and [here](https://lisk.com/documentation/beta/understand-blockchain/index.html#frontend-ui-integration).
- Identify any other location which might require the addition of the aforementioned links.

#### Affected branch / URL
`development`

#### Page location
`integrate-blockchain/aggregate-blockchain-data/index.adoc`
`run-blockchain/index.adoc#lisk-service-nodes`
`understand-blockchain/index.adoc#frontend-ui-integration`

### Depends on
https://github.com/LiskHQ/lisk-docs/issues/1585

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1836

### CODE REVIEW

File: docs/modules/ROOT/pages/understand-blockchain/index.adoc [hunk 1]
```diff
@@ -370,8 +373,9 @@ In this case, we recommend using the enriched API of Lisk Service as middleware
...
 // Another example is the UI for the Hello World app, detailed in the guide: xref:{url_integrate_ui}[].
 
-. **[recommended]** For later requirements in a production environment, we highly recommend using xref:{url_lisk_service_intro}[Lisk Service], which will aggregate the data from the blockchain network and other 3rd party sources and will provide an enriched API with a lot more endpoints and query options for the UI.
+. **[recommended]** For later requirements in a production environment, we highly recommend using xref:{url_lisk_service_intro}[Lisk Service], which will xref:{url_aggregate_lisk_service}[aggregate the data from the blockchain network] and other 3rd party sources and will provide an enriched API with a lot more endpoints and query options for the UI.
 With Lisk Service, you can also create a custom service as per your business requirements to support various UI projects, such as mobile and web applications. 
+You can also extend existing microservices; for more information, see xref:{url_extend_indexer}[].
 The frontend can then request this data via API requests from Lisk Service.
```
Comment (sameersubudhi) [inline docs/modules/ROOT/pages/understand-blockchain/index.adoc]:
```suggestion
. **[recommended]** For later requirements in a production environment, we highly recommend using xref:{url_lisk_service_intro}[Lisk Service], a middleware that xref:{url_aggregate_lisk_service}[aggregates information from the blockchain network] and other 3rd party sources to provide an extensive and enriched API.
The frontend products can access and utilize such information both over HTTP and WS.
+
With Lisk Service, you can also create a custom service as per your business requirements to support various UI projects, such as mobile and web applications.
You can also extend existing microservices; for more information, see xref:{url_extend_indexer}[].
```

File: docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc [hunk 2]
```diff
@@ -372,6 +373,8 @@ curl --location --request POST 'https://service.lisk.com/api/v3/validator/valida
 --
 =====
 
+For more information about integrating Lisk Service to a node, see xref:{url_aggregate_lisk_service}[].
```
Comment (sameersubudhi) [inline docs/modules/ROOT/pages/integrate-blockchain/connecting-endpoints.adoc]:
```suggestion
For more information about integrating Lisk Service with a Lisk application node, see xref:{url_aggregate_lisk_service}[].
```

**Review by sameersubudhi — APPROVED** [review]
Please consider the following 🙏

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Pull request merged**